pilot/pkg/xds/discovery.go

```diff
@@ -154,6 +154,9 @@ type DiscoveryServer struct {
 	// ClusterAliases are aliase names for cluster. When a proxy connects with a cluster ID
 	// and if it has a different alias we should use that a cluster ID for proxy.
 	ClusterAliases map[cluster.ID]cluster.ID
+
+	// Indicates this is running as an Sds Server.
+	SdsServer bool
```

Comment:
i donot think this is a good mark. I am thinking not add any filed here

Reply:
Any reason why you think we should add any fields here?

Do you have any alternatives to identify otherwise? we can drive this by req.typeURL if you think that is better - I can change.

Reply:
Sorry we can not do by typeURL at the place where we apply rate limit (new stream init)

Reply:
+1, can we just make the rate limit configurable and make it MaxInt for SDS?